README.md

```diff
@@ -1,20 +1,36 @@
-[![Android app on Google Play](http://ligi.de/img/play_badge.png)](https://play.google.com/store/apps/details?id=org.ligi.survivalmanual)
-[![Android app on FDroid](http://ligi.de/img/fdroid_badge.png)](https://f-droid.org/repository/browse/?fdid=org.ligi.survivalmanual)
-[![Android app at Amazon](http://ligi.de/img/amazon_badge.png)](https://www.amazon.com/ligi-Survival-Manual/dp/B06WW43R3T)
+# SurvivalManual
+
+[<img src="https://play.google.com/intl/en_us/badges/static/images/badges/en_badge_web_generic.png"
```

Comment:
I like all other changes - thanks for them - but I am somehow not a big fan of this one.  Some read the README as text and then this huge wall of things is more disturbing and I liked how it was before. Will merge  the PR after this change is  removed. 

Also IMHO the Header SurvivalManual does not really add much value

Reply:
Got it